test/compile/selection/interval.test.ts

```diff
@@ -794,7 +794,7 @@ describe('Interval Selections', () => {
           type: 'rect',
           clip: true,
           encode: {
-            enter: {fill: {value: 'transparent'}},
+            enter: {cursor: {value: 'move'}, fill: {value: 'transparent'}},
```

Comment:
Add tests that check that we don't add a cursor when 

- it's already defined
- or the selection cannot be moved (iirc Arvind made a commend about that somewhere but I can't find it right now)